lib/Minify/Env.php

```diff
@@ -78,8 +78,7 @@ public function get($key = null)
      */
     protected function computeDocRoot(array $server)
     {
-        if (empty($server['SERVER_SOFTWARE'])
-                || 0 !== strpos($server['SERVER_SOFTWARE'], 'Microsoft-IIS/')) {
+        if (isset($server['SERVER_SOFTWARE']) && 0 !== strpos($server['SERVER_SOFTWARE'], 'Microsoft-IIS/')) {
             throw new InvalidArgumentException('DOCUMENT_ROOT is not provided and could not be computed');
```

Comment:
We should also throw if the result of this calculation is "" or "/". In CLI, I think both SCRIPT_FILENAME and SCRIPT_NAME will be the same.